debian/postinst

```diff
@@ -14,6 +14,11 @@ if ! getent passwd thelounge >/dev/null; then
 		--gecos "System user for The Lounge (IRC client)"
 fi
 
+# TODO: Remove this in a few versions after v3 to help transitioning
+if [ -f /etc/lounge/config.js ] && [ ! -d /etc/thelounge ]; then
+	mv /etc/lounge /etc/thelounge
```

Comment:
add an `echo` that it's being moved?

Reply:
Sure I can do that. I didn't know if it was an appropriate thing to do.